### PR TITLE
Add omittedWords to WordInfo popover

### PR DESCRIPTION
**Pull request by fsimonjetz**
Implements omitted words part of [Tweak WordInfo popover](https://trello.com/c/Q27h9cQd)

### CODE REVIEW

File: src/corpus/domain/LineVariant.test.ts [hunk 1]
```diff
@@ -1,85 +1,22 @@
+import {
```
Comment (fsimonjetz) [inline src/corpus/domain/LineVariant.test.ts:1]: The test tokens were all very similar. I generalized them into a number of test cases and extracted them into a separate file.

File: src/transliteration/ui/LineGroup.ts [hunk 2]
```diff
@@ -12,9 +12,24 @@ export interface LineInfo {
   textService: TextService
 }
 
+function createLineTokens(
```
Comment (fsimonjetz) [inline src/transliteration/ui/LineGroup.ts:15]: Extracts aligned tokens, including the omitted ones. The order of the result doesn't matter as it's sorted later on.

File: src/transliteration/ui/line-tokens.tsx [hunk 3]
```diff
@@ -65,6 +65,7 @@ export class LineToken {
   token: LemmatizableToken
   lemma: DictionaryWord[] | null = null
   siglum: string | null = null
+  type = 'LineToken'
```
Comment (khoidt) [inline src/transliteration/ui/line-tokens.tsx:68]: `type: string = 'LineToken'`

File: src/transliteration/ui/LineLemmasContext.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React, { useContext } from 'react'
 import DictionaryWord from 'dictionary/domain/Word'
-import { LineToken } from './line-tokens'
+import { OneOfLineToken } from './line-tokens'
```
Comment (khoidt) [inline src/transliteration/ui/LineLemmasContext.ts:3]: Better use full paths, i.e. `'transliteration/ui/LineLemmasContext/line-tokens'`.

**Review by khoidt — APPROVED** [review]

File: src/transliteration/ui/line-tokens.tsx (hunk 3 above)
Comment (fsimonjetz) [reply src/transliteration/ui/line-tokens.tsx:68]: Normally yet, but in this case the linter complains `Type string trivially inferred from a string literal, remove type annotation`.

File: src/transliteration/ui/LineLemmasContext.ts (hunk 4 above)
Comment (fsimonjetz) [reply src/transliteration/ui/LineLemmasContext.ts:3]: That was an auto-import for sure. If we prefer absolute imports, maybe we should update the config so auto-imports are absolute by default.

**Pull request merged**